docs/configuration.md

```diff
@@ -137,6 +137,59 @@ jtd.addEvent(toggleDarkMode, 'click', function(){
 
 See [Customization]({{ site.baseurl }}{% link docs/customization.md %}) for more information.
 
+## Callouts
+
+To use this feature, you need to configure a `color` and (optionally) `title` for each kind of callout you want to use, e.g.:
+
+```yaml
+callouts:
+  warning:
+    title: Warning
+    color: red
+```
+
+This uses the color `$red-000` for the background of the callout, and `$red-300` for the title and box decoration.[^dark] You can then style a paragraph as a `warning` callout like this:
+
+```markdown
+{: .warning }
+A paragraph...
+```
+
+[^dark]:
+    If you use the `dark` color scheme, this callout uses `$red-300` for the background, and `$red-000` for the title. 
+
+The colors `grey-lt`, `grey-dk`, `purple`, `blue`, `green`, `yellow`, and `red` are predefined; to use a custom color, you need to define its `000` and `300` levels in your SCSS files. For example, to use `pink`, add the following to your `_sass/custom/custom.scss` file:
+
+```scss
+$pink-000: #f77ef1;
+$pink-100: #f967f1;
+$pink-200: #e94ee1;
+$pink-300: #dd2cd4;
+```
```

Comment:
With the new addition in this PR, this is implicitly done (until the user overrides `custom.scss`), right? Or am I missing something?

Reply:
Those shades of pink were defined here in [`_sass/custom/custom.scss`](https://github.com/pdmosses/just-the-docs/blob/combination/_sass/custom/custom.scss) only for use in [callouts.md](https://github.com/pdmosses/just-the-docs/blob/combination/_tests/color/callouts.md), which is located in `_tests`. When we move all the tests to a new repo, the motivation for choosing these particular shades of pink will disappear, and `_sass/custom/custom.scss` should simultaneously revert to its usual blank state. (If it didn't, we'd need to mention and motivate the contents of the file.)

Reply:
Understood, thanks!